### PR TITLE
Feature/create feedbacks layout and get skills from back-end

### PR DESCRIPTION
**Pull request by svtslv17**
image

### CODE REVIEW

File: src/components/candidateContent/candidateInfo/CandidateInfo.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { getFieldLabel } from '../../../utils';
 
 import { tableCandidateInfoFields } from '../../../constants/tableCandidateInfoFields';
 
-export const CandidateInfo = (props) => {
+const CandidateInfo = (props) => {
```
Comment (itsnotpolza) [inline src/components/candidateContent/candidateInfo/CandidateInfo.jsx:16]: why did you change the export to default?

Comment (svtslv17) [reply src/components/candidateContent/candidateInfo/CandidateInfo.jsx:16]: We export everywhere by default. So that everyone has the same. For consistency ...

**Review by itsnotpolza — APPROVED** [review]

**Review by oleksiuk-ol — APPROVED** [review]
